### ISSUE TITLE
[Accessibility]: Alternate text or representation for charts

### ISSUE THREAD

**Issue by psurana2** [carbon-design-system/carbon-charts#1860]
### Application/Team

Guardium Insights

### Environment

Browser

### What happened? What did you expect to happen instead?

1. Adding sort of an alternate text label for chart representations such as donut chart
2. Chart content needs to be accessible by keyboard 

### What WCAG 2.1 checkpoint does the issue violate?

Manual

### Version

^1.15.6

### What priority level would this issue be in your opinion?

P2
